src/it/update-line-separator-test-mojo/src/main/java/crlf/org/codehaus/mojo/license/CrLf.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * #%L
+ * License Test :: do NOT update!
+ * %%
+ * Copyright (C) 2010 Tony
+ * %%
+ * Fake to be removed!
+ * #L%
+ */
+
+/**
+ * this class owns a header, only license will be updated.
+ *
+ * @author tchemit dev@tchemit.fr
+ * @since 1.0
+ */
+package org.codehaus.mojo.license;
+
+public class Lf
```

Comment:
Even it is test file should follow java conventions, class should the same name as file.
Please also check other test files.